src/TreeView/useTypeahead.ts

```diff
@@ -18,6 +18,46 @@ export function useTypeahead({containerRef, onFocusChange}: TypeaheadOptions) {
     onFocusChangeRef.current = onFocusChange
   }, [onFocusChange])
 
+  // Focus the closest element that matches the search value
+  const focusSearchValue = React.useCallback(
+    (searchValue: string) => {
+      // Don't change focus if the search value is empty
+      if (!searchValue) return
+
+      if (!containerRef.current) return
+      const container = containerRef.current
+
+      // Get focusable elements
+      const elements = Array.from(container.querySelectorAll('[role="treeitem"]'))
+        // Filter out collapsed items
+        .filter(element => !element.parentElement?.closest('[role=treeitem][aria-expanded=false]'))
```

Comment:
```suggestion
      // Get focusable elements
      const elements = Array.from(container.querySelectorAll(':not([aria-expanded=false]) [role="treeitem"]'))
```

Would this alternative end up producing the same result as the original or would they be different sets? Just an idea if we wanted to do this in one pass

Reply:
I don't think we need to filter at all. Rows whose parents are collapsed don't get rendered. I think they used to be at one point, but that was changed. It seems to behave correctly with the filter removed.